runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java

```diff
@@ -87,7 +88,7 @@ Duration getRespawnTime(int region)
 				return region == MINING_GUILD ? Duration.ofMinutes(6) : super.respawnTime;
 			}
 		},
-	ORE_VEIN(Duration.ofSeconds(MiningOverlay.ORE_VEIN_MAX_RESPAWN_TIME), 150),
+	ORE_VEIN(Duration.ofSeconds(MiningOverlay.ORE_VEIN_MAX_RESPAWN_TIME), 150, ORE_VEIN_26661, ORE_VEIN_26662, ORE_VEIN_26663, ORE_VEIN_26664),
```

Comment:
This doesn't seem right. Not sure why the existing code is as it is but currently it adds when the depleted ore veins spawn, and this would in addition add when the non-depleted ones despawn.. did you test this at mlm?